chain-signatures/node/src/protocol/mod.rs

```diff
@@ -327,9 +320,7 @@ impl MpcSignProtocol {
 
             let message_time = Instant::now();
             if let Err(err) = state.handle(&self, &mut queue).await {
-                tracing::info!("protocol unable to handle messages: {err:?}");
-                tokio::time::sleep(Duration::from_millis(100)).await;
-                continue;
```

Comment:
Is this intentional?

Reply:
yup, erroring out on messages should have no effect whatsoever so we don't need to do continue